packages/okta-react/package.json

```diff
@@ -38,8 +38,8 @@
     "prop-types": "^15.5.10"
   },
   "peerDependencies": {
-    "react": ">=15",
-    "react-router": ">=4"
+    "react": ">=16.8.0",
+    "react-router-dom": ">=5.1.0"
```

Comment:
Should we handle console warning if other router been selected by dev? Also if I remember correctly `npm2` will get peerDependencies installed automatically, do we need to handle this scenario?

Reply:
I don't believe we support npm2 - that was quite a while ago.

I did some research, and there doesn't appear to be an option.  `optional` modules are installed if available, and `peerDependencies` are warnings.  The common approach seems to be to put in it `peerDependencies` and document.